site/docs/v1/tech/apis/users.adoc

```diff
@@ -490,6 +490,14 @@ Password hashes can be imported into FusionAuth. This allows users to transparen
 
 You can also learn more about hashes in the link:/docs/v1/tech/reference/password-hashes[password hashing reference], and you can link:/docs/v1/tech/plugins/custom-password-hashing[implement your own custom hash] as well.
 
+=== Encoding
+
+Unless you are using a custom plugin that does not work with base64 encoding, the password hash should be base64 encoded.
+
+It is recommended to use a base64 encoded string, but if importing hashes from a legacy system that uses base16, base32, or another encoding, import the hash in the same format the plugin produces.
+
+The standard FusionAuth password hashing schemes require the password hash to be a base64 encoded string.
```

Comment:
Should we also mention, that it is an option to re-encode your hash into base64? This is maybe moot since it is just an extra step.